code/__HELPERS/botany_log.dm

```diff
@@ -0,0 +1,6 @@
+//Helpers for botany logging
+/proc/get_plant_stats(obj/item/seeds/plant, obj/item/container)
+	. = "\n==Plant Stats==\nName: [plant.plantname]\n[container?"Last Fingerprint: [container.fingerprintslast]\n":""]Type: [plant.type]\n==GENES=="
+	for(var/datum/plant_gene/gene in plant.genes)
+		. += "\nGENE <[gene.get_name()]>"
```

Comment:
I'd reformat this somehow

Reply:
How? Put the \n after the GENE?

Reply:
Show pic how it would look in notepad 😄

Reply:
There is a log file generated in the pr desc

Reply:
I like it! But oh boy that's gonna be a big log file :D